exp/api/v1beta1/gcpmanagedcontrolplane_webhook.go

```diff
@@ -130,6 +130,13 @@ func (r *GCPManagedControlPlane) ValidateUpdate(oldRaw runtime.Object) (admissio
 		)
 	}
 
+	if !cmp.Equal(r.Spec.NetworkPolicy, old.Spec.NetworkPolicy) {
```

Comment:
With network polices can you do this:

- create the cluster without a network policy and then enable a policy at a later date

Just wanted to double check that.